src/index.html

```diff
@@ -856,6 +857,25 @@ <h3 class="about"><a href="//www.ebi.ac.uk/about">About us</a></h3>
 
         }
 
+        function loadGoogleAnalytics() {
+            var trackingID;
+            if (document.location.hostname == 'www.ebi.ac.uk') {
+                trackingID = 'UA-82688737-1';
+            } else if (document.location.hostname == 'wwwdev.ebi.ac.uk') {
+                trackingID = 'UA-82375858-1';
+            } else {
+                trackingID = 'UA-82386453-1';
+            }

```

Comment:
this shouldn't be a problem we can add a filter in the google analytics. 
ex: https://blog.kissmetrics.com/protect-analytics-from-hacking/